overlay.d/05core/usr/lib/dracut/modules.d/40ignition-ostree/ignition-ostree-sysusers.service

```
@@ -0,0 +1,17 @@
+[Unit]
+Description=Populate OSTree sysusers
```

For consistency, let's make it e.g.

```suggestion
Description=Ignition OSTree: Create System Users
```

?